zproject/urls.py

```diff
@@ -323,6 +324,7 @@
     # Endpoints for syncing drafts.
     rest_path("drafts", GET=fetch_drafts, POST=create_drafts),
     rest_path("drafts/<int:draft_id>", PATCH=edit_draft, DELETE=delete_draft),
+    rest_path("remind_me", POST=create_remind_me_message_backend),
```

Comment:
I think we want `reminders` for this; posted https://chat.zulip.org/#narrow/stream/378-api-design/topic/Reminde.20me.20of.20message.20in.20.3Ctime.3E.20API.20.2321810/near/1660565 on that point for discussion with the right group.